_data/nav.yml

```diff
@@ -0,0 +1,54 @@
+primary:
+  - href: /
+  - href: /getting-started/
+  - href: /design-principles/
+  - href: /about-our-work/
+  - href: /page-templates/
+
+mobile:
+  - href: /
+    collapsible: true
+  - href: /getting-started/
+    collapsible: true
+  - href: /design-principles/
+  - href: /about-our-work/
+    collapsible: true
+  - href: /page-templates/
+
+UI components:
+  - text: Overview
+    href: /
+  - href: /typography/
+  - href: /colors/
+  - href: /accessibility/
+  - href: /grids/
+  - href: /buttons/
+  - href: /labels/
+  - href: /tables/
+  - href: /alerts/
+  - href: /accordions/
+  - href: /form-controls/
+  - href: /form-templates/
+  - href: /search-bar/
+  - href: /sidenav/
+  - href: /headers/
+  - href: /footers/
+
+Getting started:
+  - text: Overview
+    href: /getting-started/
+  - href: /getting-started/developers/
+  - href: /getting-started/designers/
+  - href: /download/
```

Comment:
Ok. Thanks for clarifying. When I go to `download` page "Getting Started" is not highlighted in the top nav. Not sure if that is intentional. See screenshot below:

![draft_u_s__web_design_standards_documentation___download_code_and_design_files](https://cloud.githubusercontent.com/assets/955558/22902861/9ae12ae8-f204-11e6-83cb-37149f31dcf5.png)